pic-sure-hpds-ui/src/main/webapp/picsureui/header/header.hbs

```diff
@@ -14,28 +14,37 @@
     </div>
     {{/not_empty}}
 
-    <div class="col-md-2 dropdown btn btn-default header-btn super-admin-visible" id="super-admin-dropdown"  {{#not_contains privileges 'SUPER_ADMIN'}}style="display: none;"{{/not_contains}}>
-        <span class="dropdown-toggle col-md-12" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Super Admin Console</span>
-        <ul class="dropdown-menu" role="menu">
-            <li class="dropdown"><a id="appMgmt-super-admin" class="header-navigation" data-href="/psamaui/applicationManagement" href="#">Applications</a></li>
-            <li class="dropdown"><a id="connMgmt-super-admin" class="header-navigation" data-href="/psamaui/connectionManagement">Connections</a></li>
-            <li class="dropdown"><a id="privMgmt-super-admin" class="header-navigation" data-href="/psamaui/privilegeManagement">Privileges</a></li>
-            <li class="dropdown"><a id="roleMgmt-super-admin" class="header-navigation" data-href="/psamaui/roleManagement">Roles</a></li>
-        </ul>
-    </div>
-    <a class="col-md-2 btn btn-default header-btn authenticated-visible header-navigation" data-href="/picsureui/queryBuilder" href="#" style="display: none;" id="query-builder-btn">Query Builder</a>
-    <a class="col-md-1 btn btn-default header-btn admin-visible header-navigation" id="admin-btn" data-href="/psamaui/userManagement" href="#"
-       {{#not_contains privileges 'ADMIN'}}style="display: none;"{{/not_contains}}>Users
-    </a>
-    <a id="user-profile-btn" class="col-md-1 btn btn-default header-btn authenticated-visible" href="#" style="display: none;">User Profile</a>
-    <div class="col-md-1 dropdown btn btn-default header-btn authenticated-visible" id="help-dropdown" style="display: none;" >
-        <span class="dropdown-toggle col-md-12" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Help</span>
-        <ul class="dropdown-menu" role="menu">
-         	{{#if videoLink}}<li class="dropdown"><a href="{{{videoLink}}}">Video Demonstration</a></li>{{/if}}
-	        {{#if pdfLink}}<li class="dropdown"><a href="{{{pdfLink}}}">User Guide</a></li>{{/if}}
-            {{#if jupyterExampleLink}}<li class="dropdown"><a href="{{{jupyterExampleLink}}}">Jupyter Examples</a></li>{{/if}}
-            {{#if helpLink}}<li class="dropdown"><a href="{{{helpLink}}}">Contact Us</a></li>{{/if}}
-    </div>
+    {{#contains privileges "SUPER_ADMIN"}}
```

Comment:
This re-write is to unify how we were handling showing/hiding tabs. Formerly it was a mix of checking user privileges when rendering, and also using css post-render to show/hide elements. Now that privileges are reliably stored in the session, we don't need that redundancy.